bookwyrm/management/commands/confirm_email.py

```diff
@@ -0,0 +1,20 @@
+""" manually confirm e-mail of user """
+from django.core.management.base import BaseCommand
+
+from bookwyrm import models
+
+
+class Command(BaseCommand):
+    """command-line options"""
+
+    help = "Manually confirm email for user"
+
+    def add_arguments(self, parser):
+        parser.add_argument("username")
+
+    def handle(self, *args, **options):
+        name = options["username"]
+        user = models.User.objects.get(localname=name)
+        user.is_active = True
```

Comment:
There are actually two fields that need to be updates - `is_active`, which you have, but `deactivation_reason` should also be nulled. For a separate feature, I wrote a user method called `reactivate`, which does both those things. I think you can use that here (just call `user.reactivate()`) instead of setting the field and saving. I'm going to update the confirm email view to do that as well.